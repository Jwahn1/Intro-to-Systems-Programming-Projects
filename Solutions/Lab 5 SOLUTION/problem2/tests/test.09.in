6 Carrots 1.25
1 Milk 5.69
2 Bread 4.99
3 Carrots 1.29
2 Milk 5.69
0 End 0.00
