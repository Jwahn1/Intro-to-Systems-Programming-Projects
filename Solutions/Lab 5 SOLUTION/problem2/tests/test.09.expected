6 Carrots 1.25
3 Milk 5.69
2 Bread 4.99
3 Carrots 1.29
